server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -205,7 +209,21 @@ public <T> T getEndpointInstance(Class<T> endpointClass) {
         JsPlugins.maybeAdd(handlers::addHandler);
         // Set up /*
         handlers.addHandler(context);
-        jetty.setHandler(handlers);
+
+        final Handler handler;
+        if (config.httpCompressionOrDefault()) {
+            final GzipHandler gzipHandler = new GzipHandler();
+            // The default of 32 bytes seems a bit small.
+            gzipHandler.setMinGzipSize(1024);
+            // The GzipHandler documentation says GET is the default, but the constructor shows both GET and POST.
+            gzipHandler.setIncludedMethods(HttpMethod.GET.asString());
```

Comment:
I'm guessing you are setting this to avoid gzipping the grpc streams (which default to POST)? That might eventually not be sufficient, grpc can technically use GET as well (though as we have it written today, it shouldn't come up).

Reply:
Correct. I've updated the comment.